ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_path_selection.schema.yml

```diff
@@ -57,9 +57,7 @@ keys:
                     keys:
                       server_profiles:
                         type: list
-                        required: true
-                        min_length: 1
-                        max_length: 2
+                        max_length: 12
```

Comment:
lets's add back these to make sure we don't get an empty list  
```
required: true
min_length: 1
```

(this won't enforce stun to be configured)